submissions/moyinoluwaAdenuga/todo-app/src/App.js

```diff
@@ -0,0 +1,132 @@
+import React, {Component} from 'react'
+import Form from './Components/form'
+import Todo from './Components/todo'
+import './App.css';
+
+
+class App extends Component {
+	state = {
+		todos: [],
+		active: 0,
+		status: 'all'
+	}
+
+	// Add new item to todo array on submit
+	addTodo = (todo) => {
+		this.setState({
+			todos: [todo, ...this.state.todos],
+			active: this.state.active + 1
+		})
+	}
+
+	// To move todoitem to completed task
+    completeTodo = (id) => {
+        let updatedTodos = this.state.todos.map((todo) => {
+            if (todo.id === id) {
+                todo.completed = !todo.completed
+                todo.active = !todo.active
+            }
+            return todo
+        })
+
+		this.setState({
+			todos: updatedTodos,
+			active: this.state.active - 1
+		})
+    }   
+
+   
+    // To move completed task back to todo item
+    incompleteTodo = (id) => {
+        let updatedTodos = this.state.todos.map((todo) => {
```

Comment:
Create a function for this and call it in the incomplete todo and complete todo @MoyinoluwaA